main.js

```diff
@@ -0,0 +1,153 @@
+      },
+      {
+        itemId: 7,
+        name: 'Bodhran',
+        brand: 'Meinl',
+        price: 129.99,
+        description: 'A frame drum for people who like to jig.',
+        details: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.',
+        origin: 'Ireland',
+        imageUrl: 'https://goo.gl/YNb74O'
+      },
+      {
+        itemId: 8,
+        name: 'TR-09',
+        brand: 'Roland',
+        price: 399,
+        description: 'A drum machine for people with no desk space.',
+        details: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.',
+        origin: 'Japan',
+        imageUrl: 'https://goo.gl/gJYUxz'
+      }
+    ]
+  },
+  details: {
+    item: null
+  }
+}
+
+var $cat = app.catalog
+var $cards = $cat.items
+
+function singleCardBuilder(brand, name, price, image) {
+  var card = document.createElement('div')
+  card.setAttribute('class', 'card')
+  var cardBody = document.createElement('div')
+  cardBody.setAttribute('class', 'card-body')
+  var $brand = document.createElement('h5')
+  $brand.textContent = brand
+  $brand.setAttribute('class', 'card-text')
+  var $name = document.createElement('h3')
+  $name.setAttribute('class', 'card-title')
+  $name.textContent = name
+  var $price = document.createElement('h6')
+  $price.setAttribute('class', 'card-text')
+  $price.textContent = 'Price: $' + price
+  var $image = document.createElement('img')
+  $image.setAttribute('class', 'card-img-top')
+  $image.setAttribute('src', image)
+  card.appendChild($image)
+  card.appendChild(cardBody)
+  cardBody.appendChild($brand)
+  cardBody.appendChild($name)
+  cardBody.appendChild($price)
+
+  return card
+}
+
+function catalogBuilder(catalog) {
+  var $container = document.createElement('div')
+  $container.setAttribute('class', 'container')
+  var $heading = document.createElement('h1')
+  $heading.textContent = 'Jamazon'
+  var $deck = document.createElement('div')
+  $deck.setAttribute('class', 'card-deck')
+  var rowDiv = document.createElement('div')
+  rowDiv.setAttribute('class', 'row no-gutters')
+  for (var i = 0; i < $cards.length; i++) {
+    var $col = document.createElement('div')
+    $col.setAttribute('class', 'col-md-3')
```

Comment:
Add more column classes to change the width percentage as the viewport becomes wider.
https://getbootstrap.com/docs/4.0/layout/grid/#mix-and-match